source/blog/2016-04-27-App-Development-on-OpenShift-using-ADB.md

```diff
@@ -0,0 +1,115 @@
+eap64-mysql-persistent-s2i           Application template for EAP 6 MySQL applications with persistent storage bui...   34 (16 blank)   10
+jws30-tomcat7-mysql-persistent-s2i   Application template for JWS MySQL applications with persistent storage built...   28 (11 blank)   10
+nodejs-example                       An example Node.js application with no database                                    12 (8 blank)    5
+nodejs-mongodb-example               An example Node.js application with a MongoDB database                             13 (3 blank)    7
+
+```
+
+Step 5. Let we go ahead and deploy our first app [nodejs-example](https://github.com/openshift/nodejs-ex).
+
+```
+$ ./oc new-app nodejs-example
+--> Deploying template "nodejs-example" in project "openshift" for "nodejs-example"
+     With parameters:
+      Memory Limit=512Mi
+      Git Repository URL=https://github.com/openshift/nodejs-ex.git
+      Git Reference=
+      Context Directory=
+      Application Hostname=
+      GitHub Webhook Secret=diq3I7lgSY4IQe5IMgmA7QJB77A6SSCJBjicXd6G # generated
+      Generic Webhook Secret=ICtFb7HtU0vBKU5OxFXMR8UKxIdhAG8eiT2AYlYc # generated
+      Database Service Name=
+      MongoDB Username=
+      MongoDB Password=
+      Database Name=
+      Database Administrator Password=
+--> Creating resources with label app=nodejs-example ...
+    service "nodejs-example" created
+    route "nodejs-example" created
+    imagestream "nodejs-example" created
+    buildconfig "nodejs-example" created
+    deploymentconfig "nodejs-example" created
+--> Success
+    Build scheduled for "nodejs-example", use 'oc logs' to track its progress.
+    Run 'oc status' to view your app.
+```
+
+Step 6. Check status of your deployed application.
+
+```
+$ ./oc status
+In project OpenShift sample project (sample-project) on server https://10.1.2.2:8443
+
+svc/nodejs-example - 172.30.252.169:8080
+  dc/nodejs-example deploys istag/nodejs-example:latest <-
+    bc/nodejs-example builds https://github.com/openshift/nodejs-ex.git with openshift/nodejs:0.10 
+    #1 deployed 3 minutes ago - 1 pod
+  exposed by route/nodejs-example
+
+View details with 'oc describe <resource>/<name>' or list everything with 'oc get all'.
+```
+
+Step 7. Get your application route to access from browser. (**nodejs-example-sample-project.centos7-adb.10.1.2.7.xip.io**)
+
+```
+$ ./oc get routes
+NAME             HOST/PORT                                                       PATH      SERVICE          TERMINATION   LABELS
+nodejs-example   nodejs-example-sample-project.centos7-adb.10.1.2.7.xip.io            nodejs-example                 app=nodejs-example,template=nodejs-example
+```
+
+So now you have a sample app running on OpenShift platform. You can do lot of experiments using [client binary](https://docs.openshift.org/latest/cli_reference/index.html) or from the browser. Give it a try and let us know your feedback/concern/issues.
```

Comment:
Should we add some text and screenshots for users who would be interested to use the webconsole. Or may we just point them to some where which tells about how to use webconsole.